R/shiny.R

```diff
@@ -2294,3 +2294,50 @@ ShinyServerTimingRecorder <- R6Class("ShinyServerTimingRecorder",
 )
 
 missingOutput <- function(...) req(FALSE)
+
+#' Insert inline Markdown
+#'
+#' This function accepts a character vector of
+#' [Markdown](https://en.wikipedia.org/wiki/Markdown)-syntax text and returns a
+#' [htmltools::tagList()] of HTML tags that may be in included in an
+#' application's UI.
+#'
+#' Prior to interpretation as Markdown, leading whitespace is trimmed from text
+#' with [glue::trim()]. This makes it possible to insert Markdown and for it to
+#' be processed correctly even when the call to `markdown()` is indented.
+#'
+#' Markdown rendering is performed by [commonmark::markdown_html()]. Additional
```

Comment:
I wonder if it's best to default to `extensions = TRUE`, since that gives gfm, which I think most people will be familiar with.